### ISSUE TITLE
Upgrade to latest Meteor API

### ISSUE THREAD

**Issue by boustanihani** [cwohlman/meteor-bootboxjs#1]
I think **meteor-addons.js** needs a small updgrade, according to:
https://github.com/meteor/meteor/blob/devel/History.md#backwards-compatible-blaze-api-changes

```UI```, ```UI.insert``` and ```UI.renderWithData``` has been deprecated. We should now use ```Blaze.render()``` and ```Blaze.renderWithData()``` instead...

Check:
- http://docs.meteor.com/#/full/blaze_render
- http://docs.meteor.com/#/full/blaze_renderwithdata

**Comment by cwohlman** [comment]
Correct, I've been seeing errors about that in my console for a while. It's on the back burner for now, but I'd certainly have time to review and merge a pull request.
